docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -319,4 +325,4 @@ For more information, refer to the [C++](../../samples/cpp/benchmark_app/README.
 - [Supported Devices](supported_plugins/Supported_Devices.md)
 
 
-[autoplugin_accelerate]: ../img/autoplugin_accelerate.png
\ No newline at end of file
+[autoplugin_accelerate]: ../img/autoplugin_accelerate.png
```

Comment:
Has this image already been added?

Reply:
I don't why this is marked as a change, it is the same as old one.

Reply:
Then it's OK. :)